### PR TITLE
wrk: unbundle LuaJIT and OpenSSL

### PR DESCRIPTION
**Pull request by cho-m**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/wrk.rb [hunk 1]
```diff
@@ -3,6 +3,14 @@ class Wrk < Formula
...
   url "https://github.com/wg/wrk/archive/4.2.0.tar.gz"
   sha256 "e255f696bff6e329f5d19091da6b06164b8d59d62cb9e673625bdcd27fe7bdad"
+  # License is modified Apache 2.0 with addition to Section 4 Redistribution:
+  #
+  # (e) If the Derivative Work includes substantial changes to features
+  #     or functionality of the Work, then you must remove the name of
+  #     the Work, and any derivation thereof, from all copies that you
+  #     distribute, whether in Source or Object form, except as required
+  #     in copyright, patent, trademark, and attribution notices.
+  license "Apache-2.0"
```
Comment (cho-m) [inline Formula/wrk.rb:13]: Need confirmation if modified Apache-2.0 can be licensed as Apache-2.0.

A couple repositories have labeled it Apache-2.0 (e.g. Alpine, MacPorts, and NixOS). Debian refers to it as `Apache-2.0~glozer`

Comment (SMillerDev) [reply Formula/wrk.rb:13]: I'd say this is a `:cannot_represent` again.

File: Formula/wrk.rb [hunk 2]
```diff
@@ -14,21 +22,21 @@ class Wrk < Formula
...
+  depends_on "luajit"
+  depends_on "openssl@3"
 
   conflicts_with "wrk-trello", because: "both install `wrk` binaries"
 
   def install
     ENV.deparallelize
     ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
-    system "make"
+    ENV.append_to_cflags "-I#{Formula["luajit"].opt_include}/luajit-2.1"
```
Comment (SMillerDev) [inline Formula/wrk.rb:33]: This seems like it should be fixed upstream

Comment (cho-m) [reply Formula/wrk.rb:33]: https://github.com/wg/wrk/pull/320 would probably fix it, but PR has been open since 2018 with no comments from upstream.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**